grepgithub.py

```diff
@@ -142,10 +142,19 @@ def fetch_grep_app(page, args):
     parser.add_argument('-flang', dest='lang_filter', help='Filter language (eg. Python,C,Java). Use comma for multiple values')
     parser.add_argument('-json', dest='json_output', action='store_true', help='JSON output')
     parser.add_argument('-o', dest='output_file', help='Output file path')
+    parser.add_argument('-m', dest='monochrome', action='store_true', help='Monochrome output')
     args = parser.parse_args()
 
     out_stream = OutStream(output_file=args.output_file)
 
+    if args.monochrome:
```

Comment:
I'm not sure if setting the variables to empty strings is the most idiomatic solution, but it works.

Reply:
Simple, clean, works - perfect for such script